src/libraries/react-calendar/components/miniature/CalendarSmall.tsx

```diff
@@ -0,0 +1,144 @@
+import { useState } from 'react'
+import {
+  useCalendarState,
+  useSetCalendarState,
+} from '../../providers/calendar-context/CalendarContext'
+import { format, getMonth, addMonths, subMonths } from 'date-fns'
+import Grid from '@mui/material/Grid'
+import Tooltip from '@mui/material/Tooltip'
+import IconButton from '@mui/material/IconButton'
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft'
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+import {
+  Section,
+  Title,
+  Navigation,
+  DayHeader,
+  Day,
+  CalendarSmallWrapper,
+} from './CalendarSmall.styled'
+import { getDatesToDisplay } from '../../utils'
+
+function CalendarSmall() {
```

Comment:
it would be great if you could change all component declaration to the next style as we use it in most of our applications:
`const CalendarSmall = () => { .... }`